src/main/java/frc/robot/commands/RotateAlgaeJoint.java

```diff
@@ -0,0 +1,13 @@
+package frc.robot.commands;
+
+import com.spikes2212.command.genericsubsystem.GenericSubsystem;
+import com.spikes2212.command.genericsubsystem.commands.MoveGenericSubsystem;
+
+import java.util.function.Supplier;
+
+public class RotateAlgaeJoint extends MoveGenericSubsystem {
```

Comment:
separate into 2 commands: "RotateAlgaeJointToTop" and "RotateAlgaeJointToBottom"